src/components/Logs/Logs.js

```diff
@@ -93,40 +93,37 @@ export default class Logs extends Component {
 
   render() {
     const { func, logs, pods } = this.props
-    if (!func) { return }
+    if (!func) {
+      return
+    }
 
     return (
       <div className='logs'>
         {pods.length === 0 && <div className='emptyPods'>{'Loading Pods...'}</div>}
         {this.renderPods()}
         <div ref='logsContainer' className='logsContainer'>
-          <br />
-          <pre>
-            {logs}
-          </pre>
+          <pre className='padding-normal'>{logs}</pre>
         </div>
       </div>
     )
   }
 
   renderPods() {
     const { pods } = this.props
-    return (
-      <div className='podsList'>
-        {pods.map(pod => this.renderPod(pod))}
-      </div>
-    )
+    return <div className='podsList'>{pods.map(pod => this.renderPod(pod))}</div>
   }
 
   renderPod(pod: Pod) {
     const { selectedPod } = this.props
     const isActive = selectedPod && selectedPod.metadata.uid === pod.metadata.uid
     const status = EntityHelper.entityStatus(pod)
     return (
-      <div key={pod.metadata.uid} className={`podItem ${isActive ? 'active' : ''}`}
-        onClick={() => this.props.onSelectPod(pod)}>
-        <IconButton className='statusIconContainer'
-          style={{ width: 18, height: 12, padding: 0 }} tooltip={status}>
+      <div
+        key={pod.metadata.uid}
+        className={`podItem padding-normal ${isActive ? 'active' : ''}`}
```

Comment:
How often do we have these kind of conditionals in the code? Maybe, it's a good idea to install [JedWatson/classnames](https://github.com/JedWatson/classnames). It's a very common dependency, so I guess it's already in the bundle.

Reply:
Only occurs 2 times, not sure it's necessary to move to a library for that :)